fluidkeys/main.go

```diff
@@ -238,6 +247,25 @@ func printSecretKeyListing(key gpgwrapper.SecretKeyListing) string {
 	return output
 }
 
+func promptForKeyToImportFromGpg(secretKeyListings []gpgwrapper.SecretKeyListing) gpgwrapper.SecretKeyListing {
+	var selectedKey int
+	for validInput := false; !validInput; {
+		rangePrompt := colour.LightBlue(fmt.Sprintf("[1-%v]", len(secretKeyListings)))
+		input := promptForInput(fmt.Sprintf(PromptWhichKeyFromGPG + " " + rangePrompt + " "))
+		if intergerSelected, err := strconv.Atoi(input); err != nil {
+			fmt.Printf("Only numbers please\n")
+		} else {
+			if (intergerSelected >= 1) && (intergerSelected <= len(secretKeyListings)) {
+				selectedKey = intergerSelected - 1
+				validInput = true
+			} else {
+				fmt.Printf("Number not available\n")
```

Comment:
![image](https://user-images.githubusercontent.com/254290/45294860-a2bb3d80-b4f4-11e8-968b-06ad89dfc006.png)

"Number not available" felt a bit raw!
* a better way of saying that might be, "please select between 1 and 3"
* it seems to be missing a fullstop
* it might be good to re-display the keys?

Reply:
👍